src/index.ts

```diff
@@ -59,6 +76,11 @@ export class SgidClient {
     } else {
       this.privateKey = privateKey
     }
+
+    /**
+     * To rectify instances where an extra '\' escape character is added to the newline
+     */
+    this.privateKey = this.privateKey.replace(/\\n/gm, '\n')
```

Comment:
nice!

Reply:
yeah i encountered this issue while setting up the react-express demo and i asked @PrawiraGenestonlia and he gave me this suggestion! :D

Reply:
Yeah. We should be handling all these edge bases to ensure smooth devX.